geospace-lib/src/main/scala/com/socrata/geospace/lib/Utils.scala

```diff
@@ -15,8 +15,8 @@ object Utils extends Logging {
   // Returns (FreeMemoryInMB, FreeMemoryPercentageInt (0-100))
   private def getFree: (Int, Int) = {
     val freeMB = Runtime.getRuntime.freeMemory / MB
-    val totalMB = Runtime.getRuntime.maxMemory / MB
-    val freePct = freeMB * Pct / totalMB
+    val total = Runtime.getRuntime.maxMemory
+    val freePct = freeMB * Pct * MB / total // not converting total now is less expensive
```

Comment:
@sokrahta my way converts on division to just multiplication otherwise you would be correct, also you got to consider the return tuple in the over all scheme which limits when conversions can take place.